cli/cmd/create.go

```diff
@@ -122,8 +122,8 @@ func getConfigFromFile(filePath string) (connectors.CreateConnectorRequest, erro
 func init() {
 	RootCmd.AddCommand(createCmd)
 
-	createCmd.PersistentFlags().StringVarP(&file, "file", "f", "", "path to the config file")
-	createCmd.MarkFlagFilename("file")
+	createCmd.PersistentFlags().StringVarP(&input, "input", "i", "", "path to the config file")
```

Comment:
createCmd.PersistentFlags().StringVarP(&input, "input", "i", "", "path to the config file or folder")?